magda-storage-api/src/createApiRouter.ts

```diff
@@ -144,6 +145,69 @@ export default function createApiRouter(options: ApiRouterOptions) {
         }
     });
 
+    // Browser uploads
+    /**
+     * @apiGroup Storage
+     *
+     * @api {post} /v0/upload/{bucket} Request to upload files to {bucket}
+     *
+     * @apiDescription Uploads a file. Restricted to admins only.
+     *
+     * @apiParam (Request body) {string} bucket The name of the bucket to which to upload to
+     *
+     * @apiSuccessExample {string} 200 Successfully uploaded 2 files.
+     * {
+     *      "message": "Successfully uploaded 2 files.",
+     *      "etags": ["cafbab71cd98120b777799598f0d4808-1","19a3cb5d5706549c2f1a57a27cf30e41-1"]}
+     *
+     * @apiErrorExample {string} 500
+     *      Internal server error.
+     */
+    router.post(
+        "/upload/:bucket",
+        fileParser({ rawBodyOptions: { limit: "10mb" } }),
```

Comment:
Might need a higher (configurable) limit for our purposes

Reply:
Thanks for picking that up. Upload limit should be configurable now :+1: